extensions/amazon-aws/src/util/setupAws.ts

```diff
@@ -0,0 +1,10 @@
+import AWS from "aws-sdk";
+import { getPreferenceValues } from "@raycast/api";
+import { Preferences } from "../types";
+
+export default function setupAws() {
+  const preferences: Preferences = getPreferenceValues();
+  const credentials = new AWS.SharedIniFileCredentials({profile: preferences.aws_profile});
+  AWS.config.credentials = credentials;
+  AWS.config.update({  region: preferences.region });
+}
```

Comment:
Let's use hook here, to not configuring AWS on each render, also getPreferenceValues is a generic function, so it will be better to put Preferences type as generic parameter

```suggestion
export default function setupAws() {
  const preferences = getPreferenceValues<Preferences>();
  
  React.useEffect(() => {
    const credentials = new AWS.SharedIniFileCredentials({profile: preferences.aws_profile});
    AWS.config.credentials = credentials;
    AWS.config.update({  region: preferences.region });
  }, [preferences];
}
```

Reply:
1. added preferences.
2. converted this to hook complicate things, because the useEffect is running async, and we need to create the Instance of SQS or EC2 with updated config.
so I think the simple approach we can have a global init at the beginning of the command, right now it's just setupAws, WDYT?

Reply:
Yeah make sense, but I don't really know is it okay to run it outside of the react component, but I hope it is Okay

Just test it and if it works fine - great, let's do it😄

For now it looks great for me